wapitiCore/attack/mod_brute_login_form.py

```diff
@@ -0,0 +1,191 @@
+                                [self.password_parameter[0],payload[0]],
+                                [self.submit_var_name,self.submit_var_value]]
+
+                        request = Request(
+                            self.current_request_url,
+                            method="POST",
+                            post_params=data
+                        )
+
+                        try:
+                            response = self.crawler.send(request, follow_redirects=True)
+
+                        except ReadTimeout:
+                            if timeouted:
+                                continue
+
+                            self.log_orange("---")
+                            self.log_orange(Anomaly.MSG_TIMEOUT, page)
+                            self.log_orange(Anomaly.MSG_EVIL_REQUEST)
+                            self.log_orange(request.http_repr())
+                            self.log_orange("---")
+                            anom_msg = Anomaly.MSG_QS_TIMEOUT
+
+                            self.add_anom(
+                                request_id=original_request.path_id,
+                                category=Anomaly.RES_CONSUMPTION,
+                                level=Anomaly.MEDIUM_LEVEL,
+                                request=request,
+                                info=anom_msg,
+                                parameter=self.username_parameter_field[0]
+                            )
+
+                            timeouted = True
+
+                        else:
+                            if (self.check_success_auth(response.content)) and (response.content != false_response.content):
+                                self.log_red(_("Credentials found on {} ! ").format(self.current_request_url))
+                                vuln_message = _("Brute force authentication success !")
+                                log_message = _("Brute force authentication success !")
+
+                                self.add_vuln(
+                                    request_id=original_request.path_id,
+                                    category=Vulnerability.BASIC_AUT_BF,
+                                    level=Vulnerability.HIGH_LEVEL,
+                                    request=request,
+                                    info=vuln_message,
+                                    parameter=self.username_parameter_field[0]
+                                )
+
+                                self.log_red("---")
+                                self.log_red(
+                                    log_message,
+                                    self.MSG_VULN,
+                                    page,
+                                    self.username_parameter_field[0]
+                                )
+                                self.log_red(Vulnerability.MSG_EVIL_REQUEST)
+                                self.log_red(request.http_repr())
```

Comment:
Maybe to verbose, what about just keeping the "Credentials found on ..." + MSG_EVIL_REQUEST  + request.http_repr() ?